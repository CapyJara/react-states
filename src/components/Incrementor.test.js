import React from 'react';
import { shallow } from 'enzyme';
import Incrementor from './Incementor';

describe('incrementor', () => {
  it('renders incrementor', () => {
    const wrapper = shallow(<Incrementor />);
    expect(wrapper).toMatchSnapshot();
  });
});
