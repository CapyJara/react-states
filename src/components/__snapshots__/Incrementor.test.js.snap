// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`incrementor renders incrementor 1`] = `
<Fragment>
  <p>
    Current count is: 
    0
  </p>
  <button
    onClick={[Function]}
  >
    Increment
  </button>
</Fragment>
`;
